src/index.ts

```diff
@@ -277,41 +283,12 @@ export default class DeviseTokenAuthClient {
      * Configure DeviseTokenAuthClient with the given options.
      */
     constructor(params: Partial<Config> | Array<Record<string, Partial<Config>>>) {
-        // user is using multiple concurrent configs (>1 user types).
-        if (params instanceof Array && params.length) {
```

Comment:
Instead of gutting this completely, should we just comment out the body and add a specific error that says multiple configs are not currently supported?

Reply:
yeah, that makes more sense. I'll update it.

Reply:
updated ✅